src/dev-journal.js

```diff
@@ -10,6 +10,31 @@ const subtractMe = (date) => (_getDay(date)(1) ? 3 : 1)
 const prevWorkDay = (date) => (
   new Date(new Date().setDate(new Date(date).getUTCDate() - subtractMe(date)))
 ).toISOString().split('T')[0]
+const getPastDate = (subtractFrom, subtractMe) => {
+  const date = new Date(new Date().setDate(new Date(subtractFrom).getUTCDate() - (subtractMe + 2)))
```

Comment:
is the +2 to cover for a full 7-day week?

Reply:
ya, is a hacky-ish way to skip weekends since we're subtracting by 5 everytime 🌚